### PR TITLE
Fixes #763 protocol parameters should be variable in population simul…

### LINKED ISSUE #763: Add protocol parameters to Population user defined variability
That means that the flag has been set in MoBi. Parameter should be shown no matter what

### PR DESCRIPTION
**Pull request by msevestre**
…ation

### CODE REVIEW

File: src/PKSim.Core/Model/PopulationSimulation.cs [hunk 1]
```diff
@@ -58,7 +58,8 @@ public override TBuildingBlock BuildingBlock<TBuildingBlock>()
       /// <summary>
       ///    Returns all parameters that could potentially be defined as advanced parameters in a simulation 
       /// </summary>
-      public virtual IEnumerable<IParameter> AllPotentialAdvancedParameters => ParametersOfType(PKSimBuildingBlockType.Simulation | PKSimBuildingBlockType.Compound | PKSimBuildingBlockType.Event | PKSimBuildingBlockType.Formulation);
+      public virtual IEnumerable<IParameter> AllPotentialAdvancedParameters => 
+         ParametersOfType(PKSimBuildingBlockType.Simulation | PKSimBuildingBlockType.Compound | PKSimBuildingBlockType.Event | PKSimBuildingBlockType.Formulation | PKSimBuildingBlockType.Protocol);
```
Comment (msevestre) [inline src/PKSim.Core/Model/PopulationSimulation.cs]: I added the protocol parameters. 
I think this absolutely makes sense. By default none of the parameter are CanBeVariedInPopulation so they don't show up anyways. But if you go to mobi and change the flag to true, they should appear there.

The next step would be to actually always set the dose for example as variable in pop... but this does not have to be done now

Comment (Yuri05) [reply src/PKSim.Core/Model/PopulationSimulation.cs]: Unless somebody starts to vary application start times, should be ok

Comment (msevestre) [reply src/PKSim.Core/Model/PopulationSimulation.cs]: Lol yeah. In that case, selber schuld

Comment (msevestre) [reply src/PKSim.Core/Model/PopulationSimulation.cs]: Lol yeah. In that case, selber schuld

Comment (msevestre) [reply src/PKSim.Core/Model/PopulationSimulation.cs]: Lol yeah. In that case, selber schuld

**Pull request merged**